dist/injector.lite.js

```diff
@@ -0,0 +1,370 @@
+/******/ })
+/************************************************************************/
+/******/ ([
+/* 0 */
+/***/ function(module, exports, __webpack_require__) {
+
+	__webpack_require__(1);
+
+
+/***/ },
+/* 1 */
+/***/ function(module, exports) {
+
+	/* jshint browser: true, node: false */
+	/* global jQuery */
+
+	/*
+	 * Service Injector - Takes libraries names and memory namespaces and
+	 * returns are promise for them. This file is included in the header.
+	 *
+	 * There are a lot of dependencies in the application.  This function returns
+	 * a promise for them.  It also loads libraries that have not been loaded.
+	 *
+	 *
+	 */
+	window.FAI = window.FAI || {};
+	window.FAI.bundleHost = window.FAI.bundleHost || '/';
+
+	(function (NS, jQuery) {
+	    'use strict';
+	    var assets = NS.WebpackAssets || [],
+	        features = {},
+	        nsFeatures = {},
+	        getDeferredFeature,
+	        featureLoadFail,
+	        featureExecuteFail,
+	        featureExecuteSuccess,
+	        featureLoadSuccess,
+	        scanForFeature,
+	        loadUrl,
+	        getBundleNameForFeatureName;
+
+	    /**
+	     * Returns a deferred object for a given feature name.
+	     * @param {string} feature - The name of the feature.
+	     * @return {object} featurs[feature] - A deferred object.
+	     */
+
+	    getDeferredFeature = function (feature) {
+	        features = features || {};
+	        return features[feature];
+	    };
+
+	    /**
+	     * Updates the deferred object when the library has been loaded but not executed.
+	     * @param {object} deferredFeature - The deferredFeature.
+	     */
+
+	    featureExecuteSuccess = function (deferredFeature) {
+	        deferredFeature.progress({isLoaded: true, exists:true});
+	    };
+
+	    /**
+	     * Updates the deferred object when the library has could not load.
+	     * @param {object} deferredFeature - The deferredFeature.
+	     */
+
+	    featureExecuteFail = function (deferredFeature) {
+	        deferredFeature.reject({isLoaded: false, exists:true});
+	    };
+
+	    /**
+	     * Updates the deferred object when the library has loaded.
+	     * @param {object} deferredFeature - The deferredFeature.
+	     */
+
+	    featureLoadSuccess = function (deferredFeature) {
+	        deferredFeature.resolve({isLoaded: true, exists:true});
+	    };
+
+	    /**
+	     * Updates the deferred object when the library has could not load.
+	     * @param {object} deferredFeature - The deferredFeature.
+	     */
+
+	    featureLoadFail = function (deferredFeature) {
+	        deferredFeature.reject({isLoaded: false, exists:true});
+	    };
+
+	    /**
+	     * Sets up the handler that searches the DOM for resources to load.
+	     */
+
+	    scanForFeature = function () {
+	        jQuery('[data-bundle]').each(function (idx, el) {
+	            var resource = jQuery(el).data().bundle,
+	                host =  jQuery(el).data().host || '',
+	                source  =  jQuery(el).data().source || '';
```

Comment:
Phap you're right that the file won't actually be downloaded because it is already cached by the browser, but it will be requested again and then executed again. This is a problem because decorated namespaces may be overwritten.